### PR TITLE
OPENEUROPA-2394: Update footer links for 2.x.

### PR DESCRIPTION
**Pull request by sergepavle**
### Description

Please adapt the EC footer to reflect the home page merge by adapting the links in the "European Commission" column following the example provided in ECL playground.

[https://ec.europa.eu/component-library/playground/ec/?path=/story/page-structure-footer--corporate]

Target of the new link should be inherited from the current "Policies, information and services" so it should link to "https://ec.europa.eu/info/index_[language]" (token is just an example - use correct one)

Translations of the text can be found here:
```
 'en' => 'European Commission website',
 'bg' => 'Уебсайт на Европейската комисия',
 'es' => 'Sitio web de la Comisión Europea',
 'cs' => 'Internetové stránky Evropské komise',
 'da' => 'Europa-Kommissionens eget website',
 'de' => 'Website der Europäischen Kommission',
 'et' => 'Euroopa Komisjoni veebisait',
 'el' => 'Ιστότοπος της Ευρωπαϊκής Επιτροπής',
 'fr' => 'Site web de la Commission européenne',
 'ga' => 'Suíomh gréasáin an Choimisiúin Eorpaigh',
 'hr' => 'Internetske stranice Europske komisije',
 'it' => 'Sito web della Commissione europea',
 'lv' => 'Eiropas Komisijas tīmekļa vietne',
 'lt' => 'Europos Komisijos interneto svetainė',
 'hu' => 'Az Európai Bizottság honlapja',
 'mt' => 'Is-sit web tal-Kummissjoni Ewropea',
 'nl' => 'Website van de Europese Commissie',
 'pl' => 'Strona internetowa Komisji Europejskiej',
 'pt-pt' => 'Sítio Web institucional da Comissão Europeia',
 'ro' => 'Site-ul Comisiei Europene',
 'sk' => 'Internetové stránky Európskej komisie',
 'sl' => 'Spletišče Evropske komisije',
 'fi' => 'Euroopan komission verkkosivusto',
 'sv' => 'Europeiska kommissionens webbplats',
```
 
### Change log

- Added:
- Changed:
- Deprecated:
- Removed:
- Fixed:
- Security:

### Commands

```sh
[Insert commands here]

```

### CODE REVIEW

File: oe_corporate_blocks.post_update.php [hunk 1]
```diff
@@ -12,6 +12,40 @@
...
+function oe_corporate_blocks_post_update_20002(&$sandbox) {
+  $config = \Drupal::configFactory()->getEditable('oe_corporate_blocks.data.footer');
+  $config->set('about_ec_links', [
+    [
+      'href' => 'https://ec.europa.eu/info/index_en',
+      'label' => 'European Commission website',
+    ],
+  ]);
+  $config->save();
+  // We have to update configuration also for translated configs manually.
```
Comment (ademarco) [inline oe_corporate_blocks.post_update.php]: Isn't this gonna be imported when running the configuration import Drush command that we run after every deployment?

Comment (sergepavle) [reply oe_corporate_blocks.post_update.php]: Described

File: oe_corporate_blocks.post_update.php [hunk 2]
```diff
@@ -12,6 +12,31 @@
...
-  \Drupal::configFactory()->getEditable('oe_corporate_blocks.data.site_switcher')
+  \Drupal::configFactory()
+    ->getEditable('oe_corporate_blocks.data.site_switcher')
     ->delete();
 }
+
+/**
+ * Only display "European Commission website" link in footer.
+ */
+function oe_corporate_blocks_post_update_20002(&$sandbox) {
```
Comment (ademarco) [inline oe_corporate_blocks.post_update.php]: We can actually remove this update hook since the module does not yet have a release on branch 2.x..

Comment (sergepavle) [reply oe_corporate_blocks.post_update.php]: Done.

**Pull request merged**